pandocTemplate.latex

```diff
@@ -0,0 +1,453 @@
+\PassOptionsToPackage{unicode=true$for(hyperrefoptions)$,$hyperrefoptions$$endfor$}{hyperref} % options for packages loaded elsewhere
```

Comment:
Did you made i tyourself, or did you retrieve it from some emacs-org extension?

Reply:
The template I modified was just pandoc's latex template. I didn't retrieve anything from emacs-org.

Reply:
okey dokey :)